### DISCUSSION TITLE
Cannot install on Arch-based SteamOS due to 'python-hidapi' conflict with 'python-hid'

### DISCUSSION THREAD

**Discussion by cab4444** [streamdeck-linux-gui/streamdeck-linux-gui#191] [Q&A] [answered]
Hello, I've been trying to get my StreamDeck working on my Steam Deck (fun juggling those names) but am running into a dependency conflict between 'python-hidapi' and 'python-hid' which is required by [jupiter-hw-support](https://gitlab.com/evlaV/jupiter-hw-support). I did a clean install with yay. 
From some of the discussion I read on this [closed qmk issue](https://github.com/qmk/qmk_cli/issues/82), it seems both of these python packages use the same namespace, cannot be installed on the same system, and should be resolved between their maintainers.

In that case, is there anything that can be done on my end to get this working with Steam OS?

**Comment by cab4444** [comment, marked as answer]
Quick update - I was able to get it installed and running in a Docker container! It even detects my StreamDeck + with no issues. Now I'm just trying to figure out if it's possible to add any plugins.

**Reply by coolapso** [reply]
not exactly sure what you mean by plugins ... but streamdeck-linux-gui does not have a plugin system

**Reply by cab4444** [reply]
I just mean the official Elgato plugins, like for [twitch](https://apps.elgato.com/plugins/com.elgato.twitch) or [obs](https://apps.elgato.com/plugins/com.elgato.obsstudio).
But alright, I'll try to figure out how those plugins work and see what can be done. I also plan on looking into integrating the StreamDeck + knobs and touch panel since I've seen python-elgato-streamdeck does support it.

**Reply by coolapso** [reply]
No those don't work like that, for OBS I think you need an extension on obs itself to control it over webhooks / api calls ... not entirely sure. About the knobs .... there's discussions about it: https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/discussions/130

**Reply by cab4444** [reply]
Awesome, looks like some good conversation! I'm definitely interested in diving deeper into the whole plugin concept and see what potential there might be.
Regarding my use case for this discussion topic, do you think there's value in uploading my docker installation instructions here before closing? Not sure how many people could be looking for something similar.

**Reply by coolapso** [reply]
It's always valuable, you cal always add it to the [documentation here](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/tree/main/docs/installation)
